lib/dialects/mysql/query-interface.js

```diff
@@ -26,7 +26,10 @@ function removeColumn(tableName, columnName, options) {
   options = options || {};
 
   return this.sequelize.query(
-    this.QueryGenerator.getForeignKeyQuery(tableName, columnName),
+    this.QueryGenerator.getForeignKeyQuery(tableName.tableName ? tableName : {
+      tableName,
+      schema: this.sequelize.config.database
```

Comment:
This should be `config.schema`

Reply:
https://github.com/sequelize/sequelize/blob/master/lib/dialects/mysql/query-generator.js#L484
https://github.com/sequelize/sequelize/blob/master/lib/query-interface.js#L697
I think in mysql, schema can be seen as database. There is no schema option, no specific schema concept in sequelize, just use database for all.

Reply:
Got it, can you add a comment here explaining why schema is being assigned database